app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1107,6 +1107,18 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         featureFlag = FeatureFlags.historyMetadataUIFeature || isHistoryMetadataEnabled
     )
 
+    var jumpBackInFeature by featureFlagPreference(
```

Comment:
Can this be renamed to be `showRecentTabs`? Also, kDoc would be nice to have to help transition to DataStore when we gotta parse all these settings and figure out what each one does.

Reply:
What about `showRecentTabsFeature`? Follow other features conventions.